src/packet/bwe/mod.rs

```diff
@@ -231,3 +240,149 @@ impl fmt::Display for BandwidthUsage {
         }
     }
 }
+
+/// Sliding window [`SeqNo`]s storage.
+///
+/// Only remembers the last [`SIZE`] packets added.
+#[derive(Debug)]
+struct HandledPacketsTracker<const SIZE: usize> {
+    /// Recently added packets.
+    history: [Option<u16>; SIZE],
+
+    /// Queue that tracks added packets order so older packets are removed.
+    queue: VecDeque<u16>,
```

Comment:
Optimization: We only ever use the value modulo SIZE, with SIZE at 256 we can half the storage requirement here by only storing u8

Reply:
This is to account for possible collisions.